Code/Source/Lidar/PipelineGraph.cpp

```diff
@@ -13,32 +13,31 @@
  * limitations under the License.
  */
 #include <Lidar/PipelineGraph.h>
-#include <Lidar/RaycastResults.h>
 #include <Utilities/RGLUtils.h>
 #include <rgl/api/extensions/ros2.h>
 
 namespace RGL
 {
-    const std::vector<rgl_field_t> PipelineGraph::DefaultFields{ RGL_FIELD_XYZ_F32 };
+    const AZStd::array<rgl_field_t, 2> PipelineGraph::DefaultFields{ RGL_FIELD_IS_HIT_I32, RGL_FIELD_XYZ_F32 };
```

Comment:
Since C++11, you can initialize static const members right inside class definition - in header file, in place where you declare this member.